packages/account/src/Sections/Security/Passkeys/components/__tests__/passkeys-list.spec.tsx

```diff
@@ -2,7 +2,30 @@ import React from 'react';
 import { screen, render } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import PasskeysList from '../passkeys-list';
-import { mock_passkeys_list } from '../../__tests__/passkeys.spec';
+
+const passkey_name_1 = 'Test Passkey 1';
+const passkey_name_2 = 'Test Passkey 2';
+
+const mock_passkeys_list: React.ComponentProps<typeof PasskeysList>['passkeys_list'] = [
```

Comment:
why we created here copy of mock_passkeys_list instead of using from parent?

Reply:
It is failing the testcase, @amina-deriv, @Dhruv-deriv and I spent half a day debugging this. We shouldn't import rather keep in util files because it was for some reason causing the exporting file testcases to run.